src/intervals/rounding.jl

```diff
@@ -238,3 +152,5 @@ function set_interval_rounding(mode)
 
     parameters.rounding = mode  # a symbol
 end
+
+big{T}(x::Interval{T}) = convert(Interval{BigFloat}, x)
```

Comment:
Shouldn't `big` be imported from base (`Base.big`)? Otherwise, we do not add a new method...